src/Sylius/Behat/Context/Setup/ProductContext.php

```diff
@@ -533,6 +533,26 @@ public function thisProductHasOptionWithValues(ProductInterface $product, $optio
         $this->objectManager->flush();
     }
 
+    /**
+     * @Given /^(this product) has option "([^"]+)" without values$/
```

Comment:
I wonder, what if you put this definition in the previous function `thisProductHasOptionWithValues`? It should then map `...$values` to an empty array and we can get rid of this duplicated code 🎉